manifests/t/Tencent/WeChat/3.7.6.44/Tencent.WeChat.installer.yaml

```diff
@@ -3,13 +3,11 @@
 
 PackageIdentifier: Tencent.WeChat
 PackageVersion: 3.7.6.44
-MinimumOSVersion: 10.0.0.0
 InstallerType: nullsoft
 Scope: machine
 UpgradeBehavior: install
 Protocols:
 - weixin
-RequireExplicitUpgrade: true
```

Comment:
Is there a reason this is being removed?

Reply:
When WeChat is upgrading it will write the new version to the registry, so I think this property is not necessary. 🤔

Reply:
I just installed a previous version and its auto-updater can update the registry properly. No idea why WeChat is mentioned there.